docs/installation/tanka.md

```diff
@@ -0,0 +1,66 @@
+# Installing Loki with Tanka
+
+[Tanka](https://tanka.dev) is a reimplementation of
+[Ksonnet](https://ksonnet.io) that Grafana Labs created after Ksonnet was
+deprecated. Tanka is used by Grafana Labs to run Loki in production.
+
+## Prerequisites
+
+Grab the latest version of Tanka (at least version v0.5.0) for the `tk env`
+commands. Prebuilt binaries for Tanka can be found
+[here](https://github.com/grafana/tanka/releases/tag/v0.5.0).
```

Comment:
I hate 'here' links. I would say "can be found at Tanka Releases page" and point to https://github.com/grafana/tanka/releases instead.